src/Paystack.php

```diff
@@ -172,6 +176,21 @@ public function getAuthorizationUrl()
 
         return $this;
     }
+    
+     /**
+     * Get the authorization callback response
+     * In situations where Laravel serves as an backend for a detached UI, the api cannot redirect 
+     * and might need to take different actions based on the success or not of the transaction
+     * @return array
+     */
+    public function getAuthorizationResponse($data)
+    {
+        $this->makePaymentRequest($data);
+
+        $this->url = $this->getResponse()['data']['authorization_url'];
```

Comment:
Please rename `url` to something more explicit like `authorizationUrl`.

Also, please declare the `authorizationUrl` variable explicitly at the top of the class.